cdc/kv/client.go

```diff
@@ -108,6 +108,11 @@ type regionStatefulEvent struct {
 	finishedCallbackCh chan struct{}
 }
 
+type resolvedTsEvent struct {
+	resolvedTs uint64
+	regions    []*regionFeedState
```

Comment:
It's better to use value type instead of pointer type to reduce memory allocation. Because we can determine the slice length (ref: https://github.com/pingcap/tiflow/pull/7079/files#diff-31a8a1f4c2bd92516aebe3c694c8ab06eb69f88aa09a95212f1f5e81a91fcf1cR1331).